docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -189,7 +189,7 @@ For the automated certification kit to run successfully, there are currently the
 
 ## Server Certification Kit Installation
 
-Once the above environment has been set up, please download the xenserver-server-cert-kit-xs8.iso supplemental pack as provided by XenServer, and copy the ISO onto the /root directory of the Dom0 filesystem residing on the pool coordinator host. Use the following command to install server certification kit into all hosts in the pool:
+XenServer certification kit requires run with XenServer latest veriosn, ensure that XenServer is updated to the latest version before test.Once the above environment has been set up, please download the xenserver-server-cert-kit-xs8.iso supplemental pack as provided by XenServer, and copy the ISO onto the /root directory of the Dom0 filesystem residing on the pool coordinator host. Use the following command to install server certification kit into all hosts in the pool:

```

Comment:
this is not correct, as commented in another xencert thread: at the moment, the XS cert kits is released mapping to XS release. please re-word this.